README.md

```diff
@@ -140,6 +140,22 @@ func application(_ application: UIApplication, didFailToRegisterForRemoteNotific
 }
 ```
 
+### Location pushes
+
+Starting with iOS 15, Apple supports power efficient way to request location trough [location push service extension](https://developer.apple.com/documentation/corelocation/creating_a_location_push_service_extension).
+
+To use this new functionality you should obtain Apple entitlments, create location push service extension and make some tweaks in your push notifications code within your app:
+
+* Add Apple location pushes entitlments into your project ("AblyLocationPush" folder).
```

Comment:
This sentence doesn't really make any sense — what entitlements? What does the `AblyLocationPush` folder have to do with anything?

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/794e634e187b3df8f0a56ffac1243e41e592211e